assets/js/lib/network/socket.js

```diff
@@ -12,9 +17,32 @@ export const joinChannel = (channel) => {
     .receive('timeout', () => logMessage('Networking issue. Still waiting...'));
 };
 
+const refreshAuthTokenForSocket = async () => {
+  const refreshToken = getRefreshTokenFromStore();
+  if (!refreshToken) {
+    logError(
+      'could not refresh the access token for websockets, refresh token not found'
+    );
+    throw new Error('could not refresh access token for websocket connection');
```

Comment:
Yes, the socket connection will fail and the library will retry with an exponential backoff until it just stops, we don't have a way through the on error callback to dinstinguish between errors and abort completely the thing, so it will go for a couple of retries before stopping, but it's really a fatal error the app will break completely